vint.go

```diff
@@ -60,3 +60,46 @@ func mustReadByte(r io.Reader) (byte, error) {
 
 	return b, err
 }
+
+func WriteVInt(w io.Writer, n int64) error {
+	if n >= -112 && n <= 127 {
+		_, err := w.Write([]byte{byte(int8(n))})
```

Comment:
Upon further investigation, you don't need the intermediate cast if it's not a literal: https://play.golang.org/p/wG3PNMb5UZ